### PR TITLE
LocalizedHtmlString.Value doesn't include arguments

### PR DESCRIPTION
**Pull request by gqqnbig**
Clarify that Value doesn't include arguments

### CODE REVIEW

**Comment by pranavkm** [comment]
cc  @dougbu

File: src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ public LocalizedHtmlString(string name, string value, bool isResourceNotFound, p
         public string Name { get; }
 
         /// <summary>
-        /// The string resource.
+        /// The string resource. It doesn't include arguments, even if they are passed in.
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs]: Could you clarify how arguments could be passed into this property?

Comment (gqqnbig) [reply src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs]: @dougbu  how about the following two options?

It doesn't include string placeholder arguments, even if they were passed into the constructor.

It doesn't include string placeholder arguments, even if the instance was constructed with the arguments.

Comment (dougbu) [reply src/Microsoft.AspNetCore.Mvc.Localization/LocalizedHtmlString.cs]: :ok: I better understand what you're trying to say and agree the current comment is lacking. Suggest
```
The original resource string, without substitution of any constructor arguments.
```
or
```
The original resource string, prior to formatting with any constructor arguments.
```

(Hmm, I switched the order of "string resource" but that's unimportant. Up to you.)

**Comment by gqqnbig** [comment]
OK, I made the change. 

Do you want to squash the two commits?

**Comment by dougbu** [comment]
Also cherry-picked into `release/2.2` as fd80509fe

**Pull request merged**